mapwms.c

```diff
@@ -454,7 +459,7 @@ int msWMSApplyFilter(mapObj *map, int version, const char *filter,
       errorObj* ms_error = msGetErrorObj();
 
       if(ms_error->code != MS_NOTFOUND) {
-	msSetError(MS_WFSERR, "FLTApplyFilterToLayer() failed", "msWFSGetFeature()");
+	msSetError(MS_WMSERR, "FLTApplyFilterToLayer() failed", "msWFSGetFeature()");
```

Comment:
Should the function name for the error should also be corrected to `msWMSApplyFilter` ?

Reply:
You're right, thanks!